{
 "PRed": [
  {
   "PR": {
    "labels": [],
    "number": 1,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.4.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": 2,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.5.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": 3,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.5.3"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  }
 ],
 "bad": false,
 "pinning_version": "2025.02.07.19.05.24",
 "pre_pr_migrator_attempts": {
  "python313": 9
 },
 "pre_pr_migrator_status": {
  "python313": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/13370976310'>bot CI job</a>) @ main\n        <details>\n        <div align=\"left\">\n        <pre>\n        win_64_python3.10.____cpython: Rattler build errors: Command: rattler-build build --recipe /cf_feedstock_ops_dir/recipe -c file:///tmp/tmpb_7tx3rg -c conda-forge -c msys2 -c file:///tmp/tmpb_7tx3rg --target-platform win-64 --build-platform win-64 -m /tmp/tmp887pj2k_ --render-only --with-solve\nLogs:\n\u256D\u2500 Finding outputs from recipe\n \u2502 Loading variant config file: \"/tmp/tmp887pj2k_\"\n \u2502 Loading variant config file: \"/cf_feedstock_ops_dir/recipe/conda_build_config.yaml\"\n \u2502 Loading variant config file: \"/tmp/tmp887pj2k_\"\n \u2502 Found 1 variants\n \u2502 \n \u2502 Build variant: stacrs-0.5.3-py310hddca29c_1\n \u2502 \n \u2502 \u256D\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u252C\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u256E\n \u2502 \u2502 Variant         \u2506 Version          \u2502\n \u2502 \u255E\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u256A\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2561\n \u2502 \u2502 c_stdlib        \u2506 vs               \u2502\n \u2502 \u2502 channel_targets \u2506 conda-forge main \u2502\n \u2502 \u2502 cxx_compiler    \u2506 vs2019           \u2502\n \u2502 \u2502 python          \u2506 3.10.* *_cpython \u2502\n \u2502 \u2502 rust_compiler   \u2506 rust             \u2502\n \u2502 \u2502 target_platform \u2506 win-64           \u2502\n \u2502 \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2534\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u256F\n \u2502\n \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500 (took 0 seconds)\n\n \u256D\u2500 Resolving environments\n \u2502 \n \u2502 Resolving build environment:\n \u2502   Platform: win-64 [__unix=0=0, __linux=6.8.0=0, __glibc=2.50=0, __archspec=1=zen2]\n \u2502   Channels: \n \u2502    - file:///tmp/tmp88h4fsz3/conda-bld/\n \u2502    - file:///tmp/tmpb_7tx3rg/\n \u2502    - conda-forge\n \u2502    - msys2\n \u2502    - file:///tmp/tmpb_7tx3rg/\n \u2502   Specs:\n \u2502    - vs_win-64\n \u2502    - vs2019_win-64\n \u2502    - rust_win-64\n \u2502    - cargo-bundle-licenses\n \u2502\n \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500 (took 8 seconds)\nError: \n  \u00D7 Failed to resolve dependencies: failed to download https://\n  \u2502 conda.anaconda.org/conda-forge/win-64/repodata.json.zst\n  \u251C\u2500\u25B6 failed to download https://conda.anaconda.org/conda-forge/win-64/\n  \u2502   repodata.json.zst\n  \u2570\u2500\u25B6 failed to write to file `/tmp/tmpipl0na4d/rattler/cache/\n      repodata/.tmpT6mtzk`: No space left on device (os error 28)</pre><pre>win_64_python3.11.____cpython: Rattler build errors: Command: rattler-build build --recipe /cf_feedstock_ops_dir/recipe -c file:///tmp/tmpb_7tx3rg -c conda-forge -c msys2 -c file:///tmp/tmpb_7tx3rg --target-platform win-64 --build-platform win-64 -m /tmp/tmpxm1brjvp --render-only --with-solve\nLogs:\n\u256D\u2500 Finding outputs from recipe\n \u2502 Loading variant config file: \"/tmp/tmpxm1brjvp\"\n \u2502 Loading variant config file: \"/cf_feedstock_ops_dir/recipe/conda_build_config.yaml\"\n \u2502 Loading variant config file: \"/tmp/tmpxm1brjvp\"\n \u2502 Found 1 variants\n \u2502 \n \u2502 Build variant: stacrs-0.5.3-py311h49ce162_1\n \u2502 \n \u2502 \u256D\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u252C\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u256E\n \u2502 \u2502 Variant         \u2506 Version          \u2502\n \u2502 \u255E\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u256A\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2561\n \u2502 \u2502 c_stdlib        \u2506 vs               \u2502\n \u2502 \u2502 channel_targets \u2506 conda-forge main \u2502\n \u2502 \u2502 cxx_compiler    \u2506 vs2019           \u2502\n \u2502 \u2502 python          \u2506 3.11.* *_cpython \u2502\n \u2502 \u2502 rust_compiler   \u2506 rust             \u2502\n \u2502 \u2502 target_platform \u2506 win-64           \u2502\n \u2502 \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2534\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u256F\n \u2502\n \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500 (took 0 seconds)\n\n \u256D\u2500 Resolving environments\n \u2502 \n \u2502 Resolving build environment:\n \u2502   Platform: win-64 [__unix=0=0, __linux=6.8.0=0, __glibc=2.50=0, __archspec=1=zen2]\n \u2502   Channels: \n \u2502    - file:///tmp/tmp88h4fsz3/conda-bld/\n \u2502    - file:///tmp/tmpb_7tx3rg/\n \u2502    - conda-forge\n \u2502    - msys2\n \u2502    - file:///tmp/tmpb_7tx3rg/\n \u2502   Specs:\n \u2502    - vs_win-64\n \u2502    - vs2019_win-64\n \u2502    - rust_win-64\n \u2502    - cargo-bundle-licenses\n \u2502\n \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500 (took 9 seconds)\nError: \n  \u00D7 Failed to resolve dependencies: failed to download https://\n  \u2502 conda.anaconda.org/conda-forge/win-64/repodata.json.zst\n  \u251C\u2500\u25B6 failed to download https://conda.anaconda.org/conda-forge/win-64/\n  \u2502   repodata.json.zst\n  \u2570\u2500\u25B6 failed to write to file `/tmp/tmpipl0na4d/rattler/cache/\n      repodata/.tmpGmFBVA`: No space left on device (os error 28)</pre><pre>win_64_python3.12.____cpython: Rattler build errors: Command: rattler-build build --recipe /cf_feedstock_ops_dir/recipe -c file:///tmp/tmpb_7tx3rg -c conda-forge -c msys2 -c file:///tmp/tmpb_7tx3rg --target-platform win-64 --build-platform win-64 -m /tmp/tmp6gzsttxq --render-only --with-solve\nLogs:\n\u256D\u2500 Finding outputs from recipe\n \u2502 Loading variant config file: \"/tmp/tmp6gzsttxq\"\n \u2502 Loading variant config file: \"/cf_feedstock_ops_dir/recipe/conda_build_config.yaml\"\n \u2502 Loading variant config file: \"/tmp/tmp6gzsttxq\"\n \u2502 Found 1 variants\n \u2502 \n \u2502 Build variant: stacrs-0.5.3-py312h21dd274_1\n \u2502 \n \u2502 \u256D\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u252C\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u256E\n \u2502 \u2502 Variant         \u2506 Version          \u2502\n \u2502 \u255E\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u256A\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2561\n \u2502 \u2502 c_stdlib        \u2506 vs               \u2502\n \u2502 \u2502 channel_targets \u2506 conda-forge main \u2502\n \u2502 \u2502 cxx_compiler    \u2506 vs2019           \u2502\n \u2502 \u2502 python          \u2506 3.12.* *_cpython \u2502\n \u2502 \u2502 rust_compiler   \u2506 rust             \u2502\n \u2502 \u2502 target_platform \u2506 win-64           \u2502\n \u2502 \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2534\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u256F\n \u2502\n \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500 (took 0 seconds)\n\n \u256D\u2500 Resolving environments\n \u2502 \n \u2502 Resolving build environment:\n \u2502   Platform: win-64 [__unix=0=0, __linux=6.8.0=0, __glibc=2.50=0, __archspec=1=zen2]\n \u2502   Channels: \n \u2502    - file:///tmp/tmp88h4fsz3/conda-bld/\n \u2502    - file:///tmp/tmpb_7tx3rg/\n \u2502    - conda-forge\n \u2502    - msys2\n \u2502    - file:///tmp/tmpb_7tx3rg/\n \u2502   Specs:\n \u2502    - vs_win-64\n \u2502    - vs2019_win-64\n \u2502    - rust_win-64\n \u2502    - cargo-bundle-licenses\n \u2502\n \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500 (took 7 seconds)\nError: \n  \u00D7 Failed to resolve dependencies: failed to download https://\n  \u2502 conda.anaconda.org/conda-forge/win-64/repodata.json.zst\n  \u251C\u2500\u25B6 failed to download https://conda.anaconda.org/conda-forge/win-64/\n  \u2502   repodata.json.zst\n  \u2570\u2500\u25B6 failed to write to file `/tmp/tmpipl0na4d/rattler/cache/\n      repodata/.tmpaEYoOj`: No space left on device (os error 28)</pre><pre>win_64_python3.13.____cp313: Rattler build errors: Command: rattler-build build --recipe /cf_feedstock_ops_dir/recipe -c file:///tmp/tmpb_7tx3rg -c conda-forge -c msys2 -c file:///tmp/tmpb_7tx3rg --target-platform win-64 --build-platform win-64 -m /tmp/tmp0n9id5ih --render-only --with-solve\nLogs:\n\u256D\u2500 Finding outputs from recipe\n \u2502 Loading variant config file: \"/tmp/tmp0n9id5ih\"\n \u2502 Loading variant config file: \"/cf_feedstock_ops_dir/recipe/conda_build_config.yaml\"\n \u2502 Loading variant config file: \"/tmp/tmp0n9id5ih\"\n \u2502 Found 1 variants\n \u2502 \n \u2502 Build variant: stacrs-0.5.3-py313hbcc60ab_1\n \u2502 \n \u2502 \u256D\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u252C\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u256E\n \u2502 \u2502 Variant         \u2506 Version          \u2502\n \u2502 \u255E\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u256A\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2561\n \u2502 \u2502 c_stdlib        \u2506 vs               \u2502\n \u2502 \u2502 channel_targets \u2506 conda-forge main \u2502\n \u2502 \u2502 cxx_compiler    \u2506 vs2019           \u2502\n \u2502 \u2502 python          \u2506 3.13.* *_cp313   \u2502\n \u2502 \u2502 rust_compiler   \u2506 rust             \u2502\n \u2502 \u2502 target_platform \u2506 win-64           \u2502\n \u2502 \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2534\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u256F\n \u2502\n \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500 (took 0 seconds)\n\n \u256D\u2500 Resolving environments\n \u2502 \n \u2502 Resolving build environment:\n \u2502   Platform: win-64 [__unix=0=0, __linux=6.8.0=0, __glibc=2.50=0, __archspec=1=zen2]\n \u2502   Channels: \n \u2502    - file:///tmp/tmp88h4fsz3/conda-bld/\n \u2502    - file:///tmp/tmpb_7tx3rg/\n \u2502    - conda-forge\n \u2502    - msys2\n \u2502    - file:///tmp/tmpb_7tx3rg/\n \u2502   Specs:\n \u2502    - vs_win-64\n \u2502    - vs2019_win-64\n \u2502    - rust_win-64\n \u2502    - cargo-bundle-licenses\n \u2502\n \u2570\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500 (took 6 seconds)\nError: \n  \u00D7 Failed to resolve dependencies: failed to download https://\n  \u2502 conda.anaconda.org/conda-forge/win-64/repodata.json.zst\n  \u251C\u2500\u25B6 failed to download https://conda.anaconda.org/conda-forge/win-64/\n  \u2502   repodata.json.zst\n  \u2570\u2500\u25B6 failed to write to file `/tmp/tmpipl0na4d/rattler/cache/\n      repodata/.tmpkxh9vv`: No space left on device (os error 28)\n        </pre>\n        </div>\n        </details>"
 },
 "smithy_version": "3.45.4"
}